mlflow/pipelines/steps/train.py

```diff
@@ -457,6 +489,34 @@ def render_schema(inputs, title):
         else:
             run_card_tab.add_markdown("MODEL_URI", f"**MLflow Model URI:** `{model_uri}`")
 
+        # Tab 8: Best Parameters
+        if best_estimator_params:
+            tuning_params_card_tab = card.add_tab(
+                "Best Parameters",
+                "{{ SEARCH_SPACE }} " + "{{ BEST_PARAMETERS }} ",
+            )
+            tuning_params = yaml.dump(self.step_config["tuning"]["parameters"])
+            tuning_params_card_tab.add_html(
+                "SEARCH_SPACE",
+                f"<b>Tuning search space:</b> <br><pre>{tuning_params}</pre><br><br>",
+            )
```

Comment:
Can we move the tuning search space information to the HP trials section? The best parameters section is concerned with displaying the best parameters. The tuning section is concerned with displaying the search space and results at each point.